src/App.tsx

```diff
@@ -1,155 +1,105 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+
+const USER_ID = 10586;
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState<string>('');
+  const [selected, setSelected] = useState<string>('All');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hideError, setHideError] = useState<boolean>(false);
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
+  const loadTodos = async () => {
+    try {
+      setHideError(false);
+
+      const data = await getTodos(USER_ID);
+
+      if ('Error' in data) {
+        throw new Error('Error1');
+      } else {
+        setTodos(data);
+      }
+    } catch {
+      setHideError(true);
+
+      throw new Error('Error2');
+    }
+  };
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  useEffect(() => {
+    const timerId = setTimeout(() => {
+      setHideError(false);
+    }, 3000);
+
+    return () => {
+      clearInterval(timerId);
+    };
+  }, []);
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  useEffect(() => {
+    loadTodos();
+  }, []);
+
+  // eslint-disable-next-line react-hooks/rules-of-hooks
+  const filterTodo = useCallback(() => {
+    switch (selected) {
+      case 'All':
+
+        return todos;
+
+      case 'Active':
+        return [...todos].filter(todo => !todo.completed);
```

Comment:
```suggestion
        return todos.filter(todo => !todo.completed);
```
Filter is a non-mutating method. No need to create a copy before